packages/server-wallet/src/chain-service/chain-service.ts

```diff
@@ -245,6 +257,47 @@ export class ChainService implements ChainServiceInterface {
     return transactionResponse;
   }
 
+  async challenge(
+    challengeStates: SignedState[],
+    privateKey: PrivateKey
+  ): Promise<providers.TransactionResponse> {
+    if (!challengeStates.length) {
+      throw new Error('Must challenge with at least one state');
+    }
+
```

Comment:
It might be useful to add an `INFO` log here with the `challengeStates` being used

Reply:
Why INFO and not ERROR?

Reply:
I think the line I chose to comment on just happened to include 265 within the 4 lines GitHub shows. I mean to say there should be an INFO log in the empty space on line 267. It would probably also be useful to log on line 265.

Reply:
Addressed in 974b971fa